lib/portlayer/exec/handle.go

```diff
@@ -147,6 +147,19 @@ func (h *Handle) Commit(ctx context.Context, sess *session.Session, waitTime *in
 	// make sure there is a spec
 	h.SetSpec(nil)
 	cfg := make(map[string]string)
+
+	// Set timestamps based on target state
+	switch *h.State {
```

Comment:
The order of operations creates a challenge -- that's what I originally started to do, but the reconfigures on start / stop would overwrite the tether data.  So the path of least resistance was this -- we can look at it later to change, but this is a good starting point